ersilia/utils/docker.py

```diff
@@ -312,15 +312,38 @@ def container_peak(self, model_id):
                 return
 
         except docker.errors.NotFound as e:
-            print(f"Container {container.name} not found: {e}")
+            logger.debug(f"Container {container.name} not found: {e}")
```

Comment:
Can we make the logger an instance variable?

Reply:
Done!